apps/app_manager.py

```diff
@@ -170,17 +189,30 @@ def app_walk(base_dir):
     walk_results.append((base_dir, subdirs, modules))
     return walk_results
 
+
+def get_zeroapp_class_in_module(module_):
+    if 'init_app' in dir(module_):
+        return None
+    module_content = [item for item in dir(module_) if not item.startswith('__')]
+    for item in module_content:
+        class_ = getattr(module_, item)
+        try:
+            if issubclass(class_, zero_app.ZeroApp):
+                return class_
+        except Exception as e:
+            pass  # todo : check why isinstance(class_, ClassType)==False in python2
```

Comment:
```
>>> C1 = type("C1", (), {})
>>> C2 = type("C2", (C1,), {})
>>> isinstance(C2, C1)
False
>>> issubclass(C2, C1)
True
```
Yeah, weird - but [this](https://stackoverflow.com/a/18733034/1250228) seems to explain it well, basically, `isinstance` expects an object as its first argument, so it only treats objects as "instances".